Workbooks/CosmosDb/Overview/Overview.workbook

```diff
@@ -153,1089 +149,1135 @@
             "isHiddenWhenLocked": true,
             "queryType": 1,
             "resourceType": "microsoft.resourcegraph/resources"
-          }
-        ],
-        "style": "above",
-        "queryType": 1,
-        "resourceType": "microsoft.resourcegraph/resources"
-      },
-      "name": "cosmosDB account parameters",
-      "styleSettings": {
-        "margin": "15px 0 0 0"
-      }
-    },
-    {
-      "type": 1,
-      "content": {
-        "json": "{Message}\r\n_Cosmos DB accounts_\r\n<br />\r\n<br />"
-      },
-      "name": "CosmosDBAccounts",
-      "styleSettings": {
-        "margin": "15px 0 10px 10px"
-      }
-    },
-    {
-      "type": 11,
-      "content": {
-        "version": "LinkItem/1.0",
-        "style": "tabs",
-        "links": [
-          {
-            "cellValue": "selectedTab",
-            "linkTarget": "parameter",
-            "linkLabel": "Overview",
-            "subTarget": "Overview",
-            "style": "link"
           },
           {
-            "cellValue": "selectedTab",
-            "linkTarget": "parameter",
-            "linkLabel": "Failures",
-            "subTarget": "Failures",
-            "style": "link"
+            "id": "cf189bc4-17e3-4578-a115-5e544e064896",
+            "version": "KqlParameterItem/1.0",
+            "name": "hasSubscription",
+            "type": 1,
+            "value": "true",
+            "isHiddenWhenLocked": true,
+            "criteriaData": [
+              {
```

Comment:
why do you need this hasSubscription param at all?  you should be able to do this in the conditional visibility of just "subscriptions [equal to] [not set]" or "subscriptions [not equal to] [not set]" without an extra param

Reply:
that's just not inputting a value right? it will be 'not set'? I tried both ways but the conditional didn't quite seem to work.
![image](https://user-images.githubusercontent.com/7664931/75922324-a7c22a00-5e17-11ea-82d8-80f902bf3b7c.png)
The No Subscriptions group is still showing for some reason so I had to make a param

Reply:
because the name of the parameter is `Subscription`, not `Subscription_s_`:
![image](https://user-images.githubusercontent.com/10158007/75933211-3a6dc380-5e2e-11ea-9a7f-e55a28f684d1.png)

Reply:
🤕 🤦‍♂ ​​​​​​​